docs/user-guide/rlhf.rst

```diff
@@ -5,7 +5,10 @@
 Model Alignment by RLHF
 @@@@@@@@@@@@@@@@@@@@@@@
 
-For the purposes of this tutorial, we will go through the entire Reinforcement Learning from Human Feedback (RLHF) pipeline using models from the NeMo Framework. These models can include LLaMa2 or Mistral, and our scripts will function consistently across them.
+.. note::
```

Comment:
Line 15/18 fix JSON Lines format.

We have a script ready to use for processing the Anthropic-HH dataset into a JSON Lines format.